zone/npc.cpp

```diff
@@ -3691,25 +3691,25 @@ void NPC::ScaleNPC(uint8 npc_level, bool always_scale, bool override_special_abi
 bool NPC::IsGuard()
 {
 	switch (GetRace()) {
-	case RT_GUARD:
+	case Race::FreeportGuard:
 		if (GetTexture() == 1 || GetTexture() == 2)
 			return true;
 		break;
-	case RT_IKSAR_2:
+	case Race::IksarCitizen:
 		if (GetTexture() == 1)
 			return true;
 		break;
-	case RT_GUARD_2:
-	case RT_GUARD_3:
-	case RT_GUARD_4:
-	case RT_HUMAN_3:
-	case RT_HALFLING_2:
-	case RT_ERUDITE_2:
-	case RT_BARBARIAN_2:
-	case RT_DARK_ELF_2:
-	case RT_TROLL_2:
+	case Race::Felguard:
+	case Race::Fayguard:
+	case Race::VahShirGuard:
+	case Race::QeynosCitizen:
+	case Race::RivervaleCitizen:
+	case Race::EruditeCitizen:
+	case Race::HalasCitizen:
+	case Race::NeriakCitizen:
+	case Race::GrobbCitizen:
 	case OGGOK_CITIZEN:
```

Comment:
This didn't get changed with the other ones

Reply:
I only targeted certain prefixed defines in the script - can do other smaller changes manually after